StringCalculatorTests/AdditionTests.cs

```diff
@@ -0,0 +1,51 @@
+using Calculator;
+using Calculator.Services;
+using NSubstitute;
+
+namespace StringCalculatorTests
+{
+    [TestFixture]
+    public class AdditionTests
+    {
+        private INumberService _numberService;
```

Comment:
A small suggestion (based on my personal preference): suffix your mock variable names with "mock". This makes it very clear what is going to be a mock and what is a concretion.